### PR TITLE
feat: DRKVI-1 UI package, rename load method to loadSource

### PR DESCRIPTION
**Pull request by ndt080**

### CODE REVIEW

File: packages/core/src/player.ts [hunk 1]
```diff
@@ -35,6 +35,8 @@ class Player extends PlayerCore {
   }
 }
 
+type IPlayerCore = FullscreenApi & TracksApi & LifecycleApi & EngineApi & EmitterApi & PlayerCore;
```
Comment (kirillkurko) [inline packages/core/src/player.ts:38]: Is there a purpose for having in this `I` prefix? I think that such prefixes usually make little to no sense and they and can always be replaced with a better name. I feel that `PlayerCoreWithApi` (or `PlayerCoreWithApis`)  is a good name here even though it's not following all the naming rules (`With` is the problem here), but I think that being more explicit is fine here. What do you think?

**Review by kirillkurko — APPROVED** [review]
That is awesome! 🔥 I left a few really minor comments, but feel free to merge this one

Incredible job on this one 💪

File: packages/core/src/api/tracks.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import type { Player } from '@/player';
+import type { Track } from '@/types';
+
+export function withTracksApi<T extends typeof Player>(constructor: T) {
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
```
[the remark this answers is not in the document]
Comment (ndt080) [reply packages/core/src/api/tracks.ts:6]: I came up with a solution for error:
```
Error: TS2545: A mixin class must have a constructor with a single rest parameter of type 'any[]'
```

```
export interface SpreadableConstructor<T> {
  new(...args: any[]): T;
}
```

File: packages/core/src/api/tracks.ts [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  return class extends constructor {
+    _tracks: Track[] = [];
+    _trackElements: HTMLTrackElement[] = [];
+
+    loadTracks(tracks: Track[], append = false): void {
+      if (!this.$mediaEl) {
+        throw new Error('Error during track loading. The media element is not defined');
+      }
```
[the remark this answers is not in the document]
Comment (ndt080) [reply packages/core/src/api/tracks.ts:14]: Created the task on the Board 
[DRKVI-3. Implement the system of exceptions and logging](https://github.com/users/ndt080/projects/3/views/1?pane=issue&itemId=30112446)

File: packages/core/src/player.ts (hunk 1 above)
Comment (ndt080) [reply packages/core/src/player.ts:38]: Cool, I'm thinking about to get rid of it

File: packages/ui/package.json [hunk 4]
```diff
@@ -19,11 +19,13 @@
   },
   "scripts": {
     "dev": "vite",
-    "build": "tsc --emitDeclarationOnly && vite build",
+    "build": "tsc --emitDeclarationOnly && vite build --mode production",
```
[the remark this answers is not in the document]
Comment (ndt080) [reply packages/ui/package.json:22]: yes

File: packages/core/src/api/tracks.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (kirillkurko) [reply packages/core/src/api/tracks.ts]: Yeah, looks good to me

**Review by kirillkurko — APPROVED** [review]
Looks good to me - feel free to merge this one 🚀

**Pull request merged**